amd_tressfx/inc/AMD_TressFX.h

```diff
@@ -167,10 +184,18 @@ struct TressFX_HairBlob
 
 struct TressFX_SceneMesh
 {
+#ifndef VULKAN
```

Comment:
Please change this to something like

    #if AMD_TRESSFX_VULKAN
    #elif AMD_TRESSFX_DIRECT3D11
    #else
        #error
    #endif